src/neo4j/_codec/hydration/v1/temporal.py

```diff
@@ -172,10 +173,15 @@ def seconds_and_nanoseconds(dt):
         seconds, nanoseconds = seconds_and_nanoseconds(value)
         return Structure(b"f", seconds, nanoseconds, tz.key)
     else:
+        if isinstance(tz, timezone):
+            # offset of the timezone is constant, so any date will do
```

Comment:
if any date will do, why not keeping a single `tz.utcoffset(value)`?

Reply:
Because the builtin `timezone` only works with the builtin `datetime` type. But `value` might (or might not) be the drivers custom `neo4j.temporal.DateTime` type.